Jellyfin.Server/Migrations/Routines/MigrateKeyframeData.cs

```diff
@@ -63,48 +56,36 @@ public MigrateKeyframeData(
     /// <inheritdoc />
     public void Perform()
     {
-        const int Limit = 100;
-        int itemCount = 0, offset = 0, previousCount;
+        const int Limit = 5000;
+        int itemCount = 0, offset = 0;
 
         var sw = Stopwatch.StartNew();
-        var itemsQuery = new InternalItemsQuery
-        {
-            MediaTypes = [MediaType.Video],
-            SourceTypes = [SourceType.Library],
-            IsVirtualItem = false,
-            IsFolder = false
-        };
 
         using var context = _dbProvider.CreateDbContext();
+        var records = context.BaseItems.Where(b => b.MediaType == MediaType.Video.ToString() && !b.IsVirtualItem).Count();
+        _logger.LogInformation("Checking {Count} items for importable keyframe data.", records);
+
         context.KeyframeData.ExecuteDelete();
         using var transaction = context.Database.BeginTransaction();
-        List<KeyframeData> keyframes = [];
-
         do
         {
-            var result = _libraryManager.GetItemsResult(itemsQuery);
-            _logger.LogInformation("Importing keyframes for {Count} items", result.TotalRecordCount);
-
-            var items = result.Items;
-            previousCount = items.Count;
-            offset += Limit;
+            var items = context.BaseItems.FromSql($"SELECT * FROM [BaseItems] WHERE MediaType = 'Video' AND IsVirtualItem = 0 ORDER BY Id LIMIT {Limit} OFFSET {offset}").ToList();
```

Comment:
Is raw sql that much faster then using linq to query?

```
var items = context.BaseItems
                .Where(i => i.MediaType == "Video" && i.IsVirtualItem == false)
                .OrderBy(i => i.Id)
                .Skip(offset)
                .Take(Limit)
                .ToList();
```

Reply:
I forgot to push, this is already changed.